src/lib/y2partitioner/widgets/encrypt.rb

```diff
@@ -66,8 +66,16 @@ def handle(event)
         nil
       end
 
+      # @macro seeCustomWidget
       def help
-        controller.methods.map { |m| send("help_for_#{m.id}", m) }.join
+        help_header =
+          if display_encrypt_method?
+            _("The following encryption methods can be chosen:")
+          else
+            _("The following encryption method will be used:")
+          end
+
+        "<p>#{help_header}</p><ul>#{help_for_methods}</ul>"
```

Comment:
In general we are also translating the html tags, so I suggest to use something like:

```
format(_("<p>%{header}</p><ul>%{content}</ul>"), header: help_header, content: help_for_methods)
```